corehq/apps/export/models/new.py

```diff
@@ -222,7 +222,7 @@ def _transform(self, value, doc, transform_dates, domain):
         if transform_dates:
             value = couch_to_excel_datetime(value, doc)
         if self.item.transform:
-            value = TRANSFORM_FUNCTIONS[self.item.transform](value, doc)
+            value = TRANSFORM_FUNCTIONS[self.item.transform](value, doc, domain)
```

Comment:
all of these signatures would need to change: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/export/const.py#L28. we'd also need need to make sure that old exports still work which i believe gets called here: https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/couchexport/models.py#L572. there are some example tests that you could leverage in `test_get_export_file`